docs/template_plugin/tests/functional/op_reference/numeric.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+
+#include "comparison.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using ComparisonTypes = ngraph::helpers::ComparisonTypes;
+
+
+namespace reference_tests {
+namespace ComparisonOpsRefTestDefinitions {
+namespace {
+
+TEST_P(ReferenceComparisonLayerTest, NumericCompareWithHardcodedRefs) {
+    Exec();
+}
```

Comment:
After #6948 there is no need of creating another test pattern. You can delete these lines to avoid tests to be executed multiple times